src/oncall/api/v0/__init__.py

```diff
@@ -53,13 +53,14 @@ def init(application, config):
     application.add_route('/api/v0/roles', roles)
     application.add_route('/api/v0/roles/{role}', role)
 
-    from . import events, event, event_swap, event_override, event_link, events_link
+    from . import events, event, event_swap, event_override, event_link, events_link, oncall_events
     application.add_route('/api/v0/events', events)
     application.add_route('/api/v0/events/{event_id}', event)
     application.add_route('/api/v0/events/swap', event_swap)
     application.add_route('/api/v0/events/override', event_override)
     application.add_route('/api/v0/events/link', events_link)
     application.add_route('/api/v0/events/link/{link_id}', event_link)
+    application.add_route('/api/v0/oncall_events', oncall_events)
```

Comment:
You should add an option in the configs to enable this endpoint and the route should only be added if set to true. It should be disabled by default since this won't be useful to most people.

Reply:
I would rename this endpoint to /api/v0/events/bonus as the current name is confusing.